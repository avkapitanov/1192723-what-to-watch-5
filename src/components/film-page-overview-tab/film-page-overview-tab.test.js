import React from "react";
import renderer from "react-test-renderer";
import FilmPageOverviewTab from "./film-page-overview-tab";

const film = {
  id: 3,
  title: ``,
  genre: [],
  year: 0,
  poster: ``,
  posterImage: ``,
  background: ``,
  video: ``,
  previewVideoLink: ``,
  backgroundColor: ``,
  rating: 9.2,
  scoresCount: 201,
  description: `Kevin's mother struggles to love her strange child, despite the increasingly dangerous things he says and does as he grows up. But Kevin is just getting started, and his final act will be beyond anything anyone imagined.`,
  director: `Lynne Ramsay`,
  starring: [
    `Tilda Swinton`,
    `John C. Reilly`,
    `Ezra Miller`
  ],
  runTime: 0,
  isFavorite: false
};

it(`FilmPageOverviewTab component render correctly`, () => {
  const tree = renderer.create(
      <FilmPageOverviewTab
        film={film}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
