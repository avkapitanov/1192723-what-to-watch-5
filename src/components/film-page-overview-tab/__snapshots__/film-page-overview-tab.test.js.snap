// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmPageOverviewTab component render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9.2
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        201
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Kevin's mother struggles to love her strange child, despite the increasingly dangerous things he says and does as he grows up. But Kevin is just getting started, and his final act will be beyond anything anyone imagined.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Lynne Ramsay
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tilda Swinton, John C. Reilly, Ezra Miller
         and other
      </strong>
    </p>
  </div>,
]
`;
